openrtb-core/src/main/protobuf/openrtb-ext.proto

```diff
@@ -18,6 +18,67 @@ package com.iabtechlab.openrtb.v2;
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+message Transparency {
+  // Domain of the entity that applied user parameters.
+  optional string domain = 1;
+
+  // Array for platform or sell-side use of any user parameters (using the list
+  // provided by DSA Transparency Taskforce). Note; See definition and list of
+  // possible user parameters as listed here, applied consistently in both bid
+  // request and/or bid response.
+  repeated int32 params = 2;
```

Comment:
I see that it's been renamed to `dsaparams` in the doc https://github.com/InteractiveAdvertisingBureau/openrtb/blob/main/extensions/community_extensions/dsa_transparency.md#object-transparency